### DISCUSSION TITLE
Title clean up regex

### DISCUSSION THREAD

**Discussion by rosgr100** [Marekkon5/onetagger#169] [Q&A] [answered]
Hi, can you give me an example on how to use the Title clean up regex on AT, because i tried to use different "formats" and i can't make it work.

For example, if i want to remove -exclude- the (Quick Hitter) part from title while searching, what the regex should be?
Also, is it possible to use multiple regular expressions and if so, how can i separate them?

Thank you.

**Comment by Marekkon5** [comment, marked as answer]
Hello, the regex to match `(Quick Hitter)` would be `\(Quick Hitter\)` (in Regex `()` have special meaning, so `\` converts them into regular `()`). 

To match multiple you can use the or `|` character. Example: `(Extended Mix)|(Clean Mix)` will match `Extended Mix` or `Clean Mix`.

You can test your regexes at: https://regex101.com/ before running 1T.

**Comment by rosgr100** [comment]
Thanks for the answer.

So i don't need to use any regular expression syntax like `replace('\(Quick Hitter\)', '')`,  i just have to enter the words i want to remove.
That was my mistake then, i thought that you have to use some kind of a regex syntax.

Tried it and it works.

**Reply by Marekkon5** [reply]
Nope, just standard regex and anything that gets matched will be replaced.
